hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestPrefetch.java

```diff
@@ -84,6 +87,7 @@
 @Category({ IOTests.class, MediumTests.class })
 public class TestPrefetch {
   private static final Logger LOG = LoggerFactory.getLogger(TestPrefetch.class);
+  protected PrefetchExecutorNotifier prefetchExecutorNotifier;
```

Comment:
Why is this global? It should be created within the context of each individual test, so better make it local on each test method.